src/App.svelte

```diff
@@ -288,6 +294,30 @@
   		height: 3rem;
 		background-color: var(--header);
 		color: var(--background);
+		transition: 0.2s;
+		
+	}
+
+	.register-button:hover{
+		background-color: var(--accent1);
+		width:20rem;
+		padding-left: 4rem;
+		padding-right: 4rem;
+
+	}
+
+	.aboutsection{
```

Comment:
a better and cleaner way to do this would be to add "flex-wrap: wrap", instead of having the "imagebelowtext" class in addition to the "aboutsection" class

Reply:
<img width="346" alt="image" src="https://user-images.githubusercontent.com/30360015/218324318-1bfd7f97-7c11-4b0d-bd28-b6abff60b2a6.png">

add gap between the image and register now button